### PR TITLE
Snowballs do stam damage

### PR DESCRIPTION
**Pull request by Erikafox**
:cl:
balance: snowball do 8 stam damage now. instead of 12 brute. what the furk.
/:cl:

### CODE REVIEW

**Comment by Anticept** [comment]
I'd like to have a snowball and a separate *iceball*.

Sometimes you just wanna have a snowball fight and NOT send people to the medical ward.

**Comment by Erikafox** [comment]
yeah actually let me do this the not lazy way

**Comment by thgvr** [comment]
snowball with rock

**Comment by Erikafox** [comment]
i tested my code this time too it actually works

**Comment by MemeSnorfer** [comment]
💖✨

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1016,7 +1016,15 @@
...
-	throwforce = 12 //pelt your enemies to death with lumps of snow
+	throwforce = 1 //pelt your enemies to death with lumps of snow
+	var/stam_dam = 8
+
+/obj/item/toy/snowball/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	var/caught = hit_atom.hitby(src, FALSE, FALSE, throwingdatum=throwingdatum)
+	if(ishuman(hit_atom) && !caught)//if they are a carbon and they didn't catch it
+		var/mob/living/carbon/human/dorkass = hit_atom
+		dorkass.apply_damage(stam_dam, STAMINA)
```
Comment (MarkSuckerberg) [inline code/game/objects/items/toys.dm]: just append this to the throw_impact() defined below

Comment (Anticept) [reply code/game/objects/items/toys.dm]: if someone caught a thrown snowball I am going to go by the rules of dodgeball and admit defeat, they would be too powerful for me.

Comment (Erikafox) [reply code/game/objects/items/toys.dm]: woops doops

**Pull request merged**